### PR TITLE
fix: homepage logo width

### LINKED ISSUE #8308: Mobile view improvements
### Describe the issue

IETF logo doesn't seem to be resizing on the homepage.

The screenshot is from Safari on iPhone 13 mini.
Same results on Firefox (on macOS) with responsive design mode.

Image

### Code of Conduct

- [x] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by NGPixel**
Fixes #8308

### CODE REVIEW

File: ietf/static/css/ietf.scss [hunk 1]
```diff
@@ -236,9 +236,12 @@ th,
 
 // Helper to constrain the size of the main logo
 .ietflogo {
-    width: 75%;
```
Comment (rjsparks) [inline ietf/static/css/ietf.scss:239]: @larseggert Where did this .75 originally come from?

Comment (larseggert) [reply ietf/static/css/ietf.scss:239]: No idea. But that used to look correct with the old CSS. 🤷

File: ietf/static/css/ietf.scss [hunk 2]
```diff
@@ -236,9 +236,12 @@ th,
...
+    width: 100%;
     max-width: 300px;
```
Comment (rjsparks) [inline ietf/static/css/ietf.scss:240]: would it make sense to set a min-width (of, perhaps, `100px`) here as well?

Comment (limck5856) [reply ietf/static/css/ietf.scss:240]: Ok

**Review by rjsparks — APPROVED** [review]

**Pull request merged**